drupal/islandora_basic_image/islandora_basic_image.module

```diff
@@ -33,6 +33,7 @@ function islandora_basic_image_form($node, array &$form_state) {
  * Implements hook_form_FORM_ID_alter().
  */
 function islandora_basic_image_form_islandora_basic_image_node_form_alter(&$form, array &$form_state, $form_id) {
+  dpm($form);
```

Comment:
Ha! probably not! my bad.
jrp

On 11/09/2015 01:33 PM, Nick Ruest wrote:
>
> In drupal/islandora_basic_image/islandora_basic_image.module 
> <https://github.com/Islandora-CLAW/CLAW/pull/104#discussion_r44319378>:
>
> > @@ -33,6 +33,7 @@ function islandora_basic_image_form($node, array &$form_state) {
> >   * Implements hook_form_FORM_ID_alter().
> >   */
> >  function islandora_basic_image_form_islandora_basic_image_node_form_alter(&$form, array &$form_state, $form_id) {
> > +  dpm($form);
>
> Don't think we should have |dpm| call here.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Islandora-CLAW/CLAW/pull/104/files#r44319378>.
>

-- 

*Jason Peak*
Digital Projects Programmer
Libraries
Louisiana State University
office 225-578-2770
jpeak5@lsu.edu <mailto:jpeak5@lsu.edu> | lsu.edu <http://www.lsu.edu/>